debian/control

```diff
@@ -0,0 +1,62 @@
+Source: couchdb
+Section: misc
+Priority: optional
+Maintainer: CouchDB Developers <dev@couchdb.apache.org>
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9),
+               dh-exec,
+               dh-systemd (>= 1.5),
+               erlang-dev (>= 1:16.b.3),
```

Comment:
are we ok with this version? I don't know whats in upstream debians as base erlang but if its 16b03 we have to stick with it. It might be nicer to use an OTP 17.x https://gist.github.com/chewbranca/07d9a6eed3da7b490b47 is still up to date \o/.

Of note is that the 19.x have that nice thing where they actually respond to a SIGTERM. Very nice to have.